edgraph/server.go

```diff
@@ -827,7 +827,11 @@ func buildUpsertQuery(qc *queryContext) string {
 	}
 
 	qc.condVars = make([]string, len(qc.req.Mutations))
-	upsertQuery := strings.TrimSuffix(qc.req.Query, "}")
+
+        // We use String Builder for effective string concatenation
```

Comment:
In my opinion, this comment is not needed. StringBuilder should be used wherever the performance is a concern.
Shall we remove the comment please?